Creates various Raritan models

hooligan-sa opened the pull request:
Creates models for: 
DSX2-16
PX2-5205R
PX3-5292R-M11
PX3-5292R
PX3-5528V-V2
PX3-5729R

danner26 commented on device-types/Raritan/DSX2-16-1.yaml:
```diff
@@ -0,0 +1,56 @@
+---
+manufacturer: Raritan
+model: DSX2-16 Console Server
```
The model name appears to be `DSX2-16`
Please remove the "`Console Server`" part

hooligan-sa replied:
Removed

hooligan-sa replied:
Remove "-1" and committed file "DSX2-16.yaml"

danner26 commented on device-types/Raritan/PX2-5205R.yaml:
```diff
@@ -0,0 +1,39 @@
+---
+manufacturer: Raritan
+model: PX2-5205R
+slug: px2-5205r
```
All slugs should have the manufacturer's name appended to them. This should start with `raritan-`

danner26 commented on device-types/Raritan/PX3-5292R-M11.yaml:
```diff
@@ -0,0 +1,58 @@
+---
+manufacturer: Raritan
+model: PX3-5292R-M11
+slug: px3-5292r-m11
```
All slugs should have the manufacturer's name appended to them. This should start with `raritan-`

hooligan-sa replied:
Manufacturer name prepended

danner26 commented on device-types/Raritan/PX3-5292R.yaml:
```diff
@@ -0,0 +1,58 @@
+---
+manufacturer: Raritan
+model: PX3-5292R
+slug: px3-5292r
```
All slugs should have the manufacturer's name appended to them. This should start with `raritan-`

hooligan-sa replied:
Manufacturer name prepended

danner26 commented on device-types/Raritan/PX3-5729R.yaml:
```diff
@@ -0,0 +1,46 @@
+slug: px3-5729r
+part_number: PX3-5729R
+u_height: 1
+is_full_depth: false
```
Please add the `airflow` data point if you have/can find it

hooligan-sa replied:
It's a PDU, so no fans/airflow information that I'm aware of

danner26 commented on device-types/Raritan/PX3-5292R.yaml:
```diff
@@ -0,0 +1,58 @@
+slug: px3-5292r
+part_number: PX3-5292R
+u_height: 1
+is_full_depth: false
```
Please add the `airflow` data point if you have/can find it

danner26 replied:
Please add the `weight` and `weight_unit` data point if you have/can find it

danner26 commented on device-types/Raritan/PX3-5292R-M11.yaml:
```diff
@@ -0,0 +1,58 @@
+slug: px3-5292r-m11
+part_number: PX3-5292R-M11
+u_height: 1
+is_full_depth: false
```
Please add the `airflow` data point if you have/can find it

danner26 commented on device-types/Raritan/PX2-5205R.yaml:
```diff
@@ -0,0 +1,39 @@
+slug: px2-5205r
+part_number: PX2-5205R
+u_height: 0
+is_full_depth: false
```
Please add the `airflow` data point if you have/can find it

hooligan-sa replied:
It's a PDU, no fans/no airflow that I'm aware of

danner26 commented on device-types/Raritan/DSX2-16-1.yaml:
```diff
@@ -0,0 +1,56 @@
+part_number: DSX2-16
+comments: Dominion SX II Console Server
+u_height: 1
+is_full_depth: false
```
Please add the `airflow` data point if you have/can find it

hooligan-sa replied:
Was unable to find it, sorry

danner26 commented on device-types/Raritan/DSX2-16-1.yaml:
```diff
@@ -0,0 +1,56 @@
+part_number: DSX2-16
+comments: Dominion SX II Console Server
+u_height: 1
+is_full_depth: false
```
Please add the `weight` and `weight_unit` data point if you have/can find it

danner26 commented on device-types/Raritan/PX2-5205R.yaml:
```diff
@@ -0,0 +1,39 @@
+slug: px2-5205r
+part_number: PX2-5205R
+u_height: 0
+is_full_depth: false
```
Please add the `weight` and `weight_unit` data point if you have/can find it

danner26 commented on device-types/Raritan/PX3-5292R-M11.yaml:
```diff
@@ -0,0 +1,58 @@
+slug: px3-5292r-m11
+part_number: PX3-5292R-M11
+u_height: 1
+is_full_depth: false
```
Please add the `weight` and `weight_unit` data point if you have/can find it

hooligan-sa replied:
weight and weight_unit added

danner26 commented on device-types/Raritan/PX3-5528V-V2.yaml:
```diff
@@ -0,0 +1,94 @@
+slug: px3-5528v-v2
+part_number: PX3-5528V-V2
+u_height: 0
+is_full_depth: false
```
Please add the `weight` and `weight_unit` data point if you have/can find it

hooligan-sa replied:
weight and weight_unit added

danner26 commented on device-types/Raritan/PX3-5729R.yaml:
```diff
@@ -0,0 +1,46 @@
+slug: px3-5729r
+part_number: PX3-5729R
+u_height: 1
+is_full_depth: false
```
Please add the `weight` and `weight_unit` data point if you have/can find it

hooligan-sa commented on device-types/Raritan/DSX2-16-1.yaml:
```diff
@@ -0,0 +1,56 @@
+---
+manufacturer: Raritan
+model: DSX2-16 Console Server
+slug: dsx2-16-1
```
Manufacturer name prepended

hooligan-sa commented on device-types/Raritan/DSX2-16-1.yaml:
```diff
@@ -0,0 +1,56 @@
+model: DSX2-16 Console Server
+slug: dsx2-16-1
+part_number: DSX2-16
+comments: Dominion SX II Console Server
```
Added

hooligan-sa commented on device-types/Raritan/PX3-5292R-M11.yaml:
```diff
@@ -0,0 +1,58 @@
+part_number: PX3-5292R-M11
+u_height: 1
+is_full_depth: false
+comments: Rack PDU, Metered, Remote Switching, 7.4kVA, 230V, (12) C13
```
Added

hooligan-sa commented on device-types/Raritan/PX3-5292R.yaml:
```diff
@@ -0,0 +1,58 @@
+part_number: PX3-5292R
+u_height: 1
+is_full_depth: false
+comments: Rack PDU, Metered, Remote Switching, 7.4kVA, 230V, (12) C13
```
Added

hooligan-sa commented on device-types/Raritan/PX3-5729R.yaml:
```diff
@@ -0,0 +1,46 @@
+part_number: PX3-5729R
+u_height: 1
+is_full_depth: false
+comments: Rack PDU, Metered, Remote Switching, 7.4kVA, 230V, (4) C13, (4) C19
```
Added

hooligan-sa commented on device-types/Raritan/PX3-5528V-V2.yaml:
```diff
@@ -0,0 +1,94 @@
+---
+manufacturer: Raritan
+model: PX3-5528V-V2
+slug: px3-5528v-v2
```
prepended with manufacturer name

hooligan-sa commented on device-types/Raritan/PX3-5729R.yaml:
```diff
@@ -0,0 +1,46 @@
+---
+manufacturer: Raritan
+model: PX3-5729R
+slug: px3-5729r
```
manufacturer prepended

hooligan-sa commented:
Not sure why this has gone stale, can @danner26 review/advise if there's still something awaiting review?

hooligan-sa reviewed (commented):
All changes completed that I'm aware of, please advise if that's not the case

danner26 reviewed (changes requested):
Please revert the file permission on `PX3-5497V.yaml` back to `0755`

danner26 commented:
Hello, we have released a new set of tests and features to assist with properly creating definition files for the DeviceType-Library. These tests will ensure that all files conform to a standardized format and contain valid data. Over the coming weeks we will be adding more tests to handle other currently undefined cases. If you notice anything that you believe is a bug, please @ me in a comment and details what you think is incorrect.

Along with additional test cases we have added a feature that will annotate why tests fail. To access these annotations after a test has completed, please navigate to the _Files changed_ tab within your PR. At the very bottom of the page you will see a list of errors along with the filename in question and the error which was found.

With all this being said, if your PR is validated then great! Please @ me me so I can review it further! If your PR is not valid, please review the issues and correct them; in the meantime I will mark your PR as a draft. After correcting the issues and recieving a valid status, please mark the PR as ready to be reviewed. Thank you!

hooligan-sa commented:
I've removed the DSX2-16-1.yaml file and replaced it with DSX2-16.yaml
In that process I've updated the model, slug and part_number fields, and all tests on this file pass now - Please shout if there are still any issues?

hooligan-sa commented:
Hi Daniel,

I really can’t understand the requirements for those permissions, but I’ve reset them.

Regards,
Patrick Holahan

From: "Daniel W. Anner" ***@***.***>
Reply to: netbox-community/devicetype-library ***@***.***>
Date: Tuesday, 18 July 2023 at 15:13
To: netbox-community/devicetype-library ***@***.***>
Cc: Patrick Holahan ***@***.***>, Author ***@***.***>
Subject: Re: [netbox-community/devicetype-library] Creates various Raritan models (PR #1386)


@danner26 requested changes on this pull request.

________________________________

On device-types/Raritan/PX3-5497V.yaml<https://github.com/netbox-community/devicetype-library/pull/1386#discussion_r1266755910>:

Please revert the permission change you made here. If you need help with this, you are welcome to give me access to your fork and I can make the adjustment for you

—
Reply to this email directly, view it on GitHub<https://github.com/netbox-community/devicetype-library/pull/1386#pullrequestreview-1535009547>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AFA2ZS3YJA2XXZQRGWYCAZDXQ2DWVANCNFSM6AAAAAAYNLJLOA>.
You are receiving this because you authored the thread.Message ID: ***@***.***>
